src/pages/categories/Categories.js

```diff
@@ -0,0 +1,57 @@
+import { Card, Sidebar } from "components"
+import './categories.css'
+
+export const Categories = () => {
+    return(
+        <>
+        <main className="container">
+        <Sidebar />
+        <div className="main-content">
+        <Card
+            img={"./images/homepage/webdev.png"}
+            productName={"The complete 2022 Fullstack Web Developer Course"}
+            productOwner={"Arjun"}
```

Comment:
If you are giving string to props try it without {}. If it works I think then you no need to wrap it in {}